### PR TITLE
Fixes morgue unit lights and their informational paper

### PR DESCRIPTION
**Pull request by lessthnthree**
## About The Pull Request
Skyrat resprited the morgue units but never updated the documentation paper that is spawned in the morgue, which has instructions for the stock TG units.

## How This Contributes To The Skyrat Roleplay Experience
Who the hell knows what those lights on the morgue unit actually mean? The paper didn't match what was being actually reflected. This changes it so it's more clear what's going on with the morgue unit and the paper they have for training is correct.

## Changelog

:cl: LT3
fix: The lights on the morgue units are updated, and the instructions are now correct on what those colours indicate. Sorry for anyone who was hiding booze in a morgue unit!
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/aesthetics/morgue/code/morgue.dm [hunk 1]
```diff
@@ -1,2 +1,6 @@
 /obj/structure/bodycontainer
 	icon = 'modular_skyrat/modules/aesthetics/morgue/icons/morgue.dmi'
+
+/obj/item/paper/guides/jobs/medical/morgue
+	name = "morgue memo"
+	default_raw_text = "<font size='2'>Since this station's medbay never seems to fail to be staffed by the mindless monkeys meant for genetics experiments, I'm leaving a reminder here for anyone handling the pile of cadavers the quacks are sure to leave.</font><BR><BR><font size='4'><font color=magenta>Purple lights mean there's a plain ol' dead body inside.</font><BR><BR><font color=orange>Yellow lights mean there's non-body objects inside.</font><BR><font size='2'>Probably stuff pried off a corpse someone grabbed, or if you're lucky it's stashed booze.</font><BR><BR><font color=red>Red lights mean the morgue system detects the body may be able to be brought back to life.</font></font><BR><font size='2'>I don't know how that works, but keep it away from the kitchen and go yell at the geneticists.</font><BR><BR>- CentCom Medical Inspector"
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/aesthetics/morgue/code/morgue.dm]: I think I'd rather have green as the color for a body that can be revived, I think it'd make more sense than red. Other than that, I'm happy to see this.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/aesthetics/morgue/code/morgue.dm (hunk 1 above)
Comment (lessthnthree) [reply modular_skyrat/modules/aesthetics/morgue/code/morgue.dm]: My thought that a flashing red light as it audibly beeps conveys 'you did it wrong, there's someone revivable in here'

Green seems like it means you did everything fine.

Comment (lessthnthree) [reply modular_skyrat/modules/aesthetics/morgue/code/morgue.dm]: On second look, TG uses green for that too, which I was trying to make more consistent, so yeah I'll swap red to green.

**Comment by ghost** [comment]
😱 why was this merged before 24 hours so everyone could read ittttt, I was gonna say just change the light colors to match the paper!!! I planned on changing it anyways x-x

**Pull request merged**